adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java

```diff
@@ -406,19 +407,23 @@ public void testLocalSilentFailedBrokerSilentReturnErrorCannotTryWithInteractive
         cacheStore.removeAll();
     }
 
-    private ITokenCacheStore getTokenCache(final Date expiresOn, final boolean storeMRRT, final boolean storeFRT) {
+    private ITokenCacheStore getTokenCache(final Date expiresOn, final boolean storeMRRT, final boolean storeFRT, final Date extendedExpiresOn) {
         // prepare valid item in cache
         final UserInfo userInfo = new UserInfo();
         userInfo.setDisplayableId(TEST_UPN);
         userInfo.setUserId(TEST_USERID);
-
-        final String accessToken = "I am an AT";
+        final String accessToken;
+        if (extendedExpiresOn ==  null) {
+            accessToken = "I am an AT";
```

Comment:
I understand the intention you want to clearly show that if extended expires on exists, the access token is a stale token, but does it really matter to give string for the access token? The thing you verify eventually is just  the access token, and access token itself doesn't tell you it's expired or not, you get the info from the expires date stored along with the access token.